pkg/updater/resultstore/resultstore_test.go

```diff
@@ -1359,3 +1359,504 @@ func TestIncludeStatus(t *testing.T) {
 		})
 	}
 }
+
+func TestGroupInvocations(t *testing.T) {
+	cases := []struct {
+		name        string
+		tg          *configpb.TestGroup
+		invocations []*invocation
+		want        []*invocationGroup
```

Comment:
nit: Does it make sense to make the 'want' argument more compact? I think all this cares about is that each group ID has the right invoc IDs, and I think it'll be more readable.

Reply:
Good suggestion - I made the tests more compact by removing some parts which didn't matter